features/com.jboss.devstudio.core.feature/feature.xml

```diff
@@ -58,6 +58,7 @@
   <includes id="org.jboss.tools.ws.jaxrs.feature" version="0.0.0"/>
   <includes id="org.jboss.tools.ws.jaxws.feature" version="0.0.0"/>
   <includes id="org.jboss.tools.wtp.runtimes.tomcat.feature" version="0.0.0"/>
+  <includes id="org.jboss.tools.cdi.microprofile.feature" version="0.0.0"/>
```

Comment:
Move it next to other CDI features for better maintenance

Reply:
fixed.